lib/spack/spack/tengine.py

```diff
@@ -0,0 +1,117 @@
+##############################################################################
```

Comment:
I can't decide on this. On the one hand I agree that the name is not really clear. On the other hand I really dislike module names that are more than one word. Anybody with a suggestion that is both short and clear?